dataretrieval/nwis.py

```diff
@@ -758,6 +769,15 @@ def get_info(ssl_check: bool = True, **kwargs) -> Tuple[pd.DataFrame, BaseMetada
     """
     seriesCatalogOutput = kwargs.pop('seriesCatalogOutput', None)
     if seriesCatalogOutput in ['True', 'TRUE', 'true', True]:
+
+        warnings.warn(('WARNING: Beginning in late February 2024 the NWIS qw data '
+                       'endpoint will not deliver new data or updates to existing '
+                       'data. Eventually the endpoint will be retired. For updated '
+                       'information visit https://waterdata.usgs.gov.nwis/qwdata '
+                       'For additional details, see the R package vignette: '
+                       'https://doi-usgs.github.io/dataRetrieval/articles/Status.html '
+                       'If you have additional questions about the qw data service, '
+                       'email CompTools@usgs.gov.'))
```

Comment:
```suggestion
        warnings.warn(('WARNING: The NWIS qw data endpoint is retiring and '
                   'no longer receives updates. For additional information '
                   'refer to https://waterdata.usgs.gov.nwis/qwdata and '
                   'https://doi-usgs.github.io/dataRetrieval/articles/Status.html or '
                   'email CompTools@usgs.gov.'))
```

Shorter is better, but I'm not too picky

Reply:
@ehinman can you please update the message text to read 'Starting in March 2024...'

Reply:
Great, thanks for updating!